Dockerfile

```diff
@@ -60,7 +60,7 @@ RUN set -ex \
     && pip install pyOpenSSL \
     && pip install ndg-httpsclient \
     && pip install pyasn1 \
-    && pip install apache-airflow[crypto,celery,postgres,hive,jdbc,mysql]==$AIRFLOW_VERSION \
+    && pip install apache-airflow[crypto,celery,postgres,hive,jdbc,mysql,ssh]==$AIRFLOW_VERSION \
```

Comment:
Note:
`ssh` group is defined in https://github.com/apache/incubator-airflow/blob/master/setup.py#L231